src/stcal/saturation/tests/test_saturation.py

```diff
@@ -0,0 +1,43 @@
+"""
+
+Unit tests for saturation flagging
+
+"""
+
+import numpy as np
+
+from stcal.saturation.saturation import flag_saturated_pixels
+
+
+def test_basic_saturation_flagging():
```

Comment:
Will the other tests from JWST be moved to this unit tests or will they remain in the JWST unit tests?

Reply:
i think they should eventually all be moved to stcal. i put a simple test in there just so there was coverage, it will take a while to rewrite the tests without datamodels so i will do that in a separate PR